core/crdt/pncounter.go

```diff
@@ -45,6 +48,9 @@ type PNCounterDelta[T Incrementable] struct {
 	DocID     []byte
 	FieldName string
 	Priority  uint64
+	// Entropy is an added randomly generated number that ensures
+	// that each increment operation is unique.
+	Entropy int64
```

Comment:
todo: Lets rename this to `Nonce` as its more cryptographically accurate, and slightly more common

Reply:
Lol... Whoever introduced that to crypto was probably not British... It is not very polite.  If it is only slightly more common I'd suggest sticking with `Entropy`.

Reply:
FWIW `nonce` is fairly common in the cryptography world, so I have a slight pref for it.